h2o-algos/src/main/java/hex/gam/GAMModel.java

```diff
@@ -330,6 +332,18 @@ public final static double linkInv(double x, Link link, double tweedie_link_powe
     }
   }
 
+  @Override
+  protected String[][] scoringDomains(){
+    String [][] domains = _output._domains;
+    if ((_parms._family == Family.binomial || _parms._family == Family.quasibinomial ||
+            _parms._family == Family.fractionalbinomial)
+            && _output._domains[_output._dinfo.responseChunkId(0)] == null) {
+      domains = domains.clone();
+      domains[_output._dinfo.responseChunkId(0)] = BINOMIAL_CLASS_NAMES;
```

Comment:
The quasibinomial case can have a completely different domain, for example, {-1, 3.42}. it is not correct to assign {0, 1} domain by default. A few months ago I implemented MRTask `CollectDoubleDomain` to calculate the quasibinomial domain correctly. Link to the class [here]( https://github.com/h2oai/h2o-3/blob/423c907d0c8f1d31e649f8a58e1f47f76e148788/h2o-core/src/main/java/water/util/VecUtils.java#L412).

Reply:
Clever!  I used it.

Reply:
I changed it.  Thanks.